_posts/en/newsletters/2020-03-18-newsletter.md

```diff
@@ -0,0 +1,94 @@
+---
+This week's newsletter summarizes an update to a proposed standard for
+LN and includes our regular sections about notable changes to services,
+client software, and popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed watchtower BOLT updated:** Sergi Delgado Segura
+  [emailed][segura email] the Lightning-Dev mailing list an [updated
+  version][watchtower bolt] of a suggested protocol for
+  [watchtower][topic watchtowers] communication.  See [Newsletter
+  #75][news75 watchtower] for our original description of this proposal.  According to
+  Segura, the update includes details about "user accounts, payment
+  methods, and message signing."  His email also provides a list of
+  features he would like to add, with discussion about each near the end
+  of the email.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Coinbase withdrawal transactions now using batching:** Coinbase has rolled
+out [batch withdrawals][coinbase batching blog] that they estimate will reduce
+their load on the Bitcoin network by 50%. Instead of each withdrawal payment
+generating a single onchain transaction, multiple payments will be
+[combined into a single transaction][scaling payment batching] once every 10 minutes.
+
+- **Bitstamp supports bech32:** Bitstamp users can now benefit from using native
+[bech32][topic bech32] addresses after the exchange announced support for both
+[bech32 deposits and withdrawals][bitstamp bech32 blog].
+
+- **Deribit supports bech32 withdrawals:** [Deribit announced][deribit bech32
+withdrawal tweet] that its exchange users can now withdraw bitcoins to bech32
+native addresses.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16902][] changes consensus code to fix an inefficiency
+  in the parsing of `OP_IF` and related opcodes.  In legacy and segwit
+  v0 script, the inefficiency isn't believed to cause any significant
+  problems. However, the proposal for [tapscript][topic tapscript] would
+  make it possible for an attacker to use the inefficiency to create blocks with transactions
+  that could take a large amount of CPU to verify.  Fixing the
+  inefficiency now reduces the number of changes that need to be made in the
+  proposed schnorr, taproot, and tapscript soft fork.  For more
+  information, see the Bitcoin Core PR Review Club [meeting notes][club
+  #16902] about this PR and a [related PR][Bitcoin Core #18002] also
+  merged this week.
+
+- [LND #3821][] adds [anchor commitments][topic anchor outputs] for
+  LN channels and enables it by default if both participating nodes of a
```

Comment:
I'm not sure 'it' is correct here. It refers to "anchor commitments" which is plural, so suggests to me that we should use 'them'. Alternatively, "LND 3821 implements anchor commitments for enables this feature for new LN channels by default ..." or similar

Reply:
Agree, was about to suggest s/it/this feature/